Makefile

```diff
@@ -6,7 +6,7 @@ PKG := $(ORG)/rukpak
 export IMAGE_REPO ?= quay.io/operator-framework/rukpak
 export IMAGE_TAG ?= latest
 IMAGE?=$(IMAGE_REPO):$(IMAGE_TAG)
-KIND_CLUSTER_NAME ?= kind
+KIND_CLUSTER_NAME ?= rukpak
```

Comment:
Changing the default name of the cluster means we would have to do `kind delete cluster --name rukpak` or use the makefile target versus just `kind delete cluster` which I think is nicer -- is it worth the tradeoff? Not a huge deal either way though.

Reply:
Not a huge deal break for me to change this back to `kind`. Main intent was to not restart a user's existing cluster if they didn't want us to given how the `run` target works with the latest commit.

Reply:
That's fair -- we can have it be a unique name then.